spec/example_app/db/migrate/20170224010823_add_birthdate_to_customer.rb

```diff
@@ -0,0 +1,5 @@
+class AddBirthdateToCustomer < ActiveRecord::Migration[4.2]
```

Comment:
Could you set the migration to `5.1`? We've been doing that for recent migrations.